### PR TITLE
Parameter names for `n.m.client.renderer.block.model` + subpackages

### PR DESCRIPTION
**Pull request by thedarkcolour**

### CODE REVIEW

File: data/net/minecraft/client/renderer/block/model/BlockModelDefinition.mapping [hunk 1]
```diff
@@ -1,7 +1,18 @@
 CLASS net/minecraft/client/renderer/block/model/BlockModelDefinition
+	METHOD <init> (Ljava/util/List;)V
+		ARG 1 modelDefinitions
+	METHOD <init> (Ljava/util/Map;Lnet/minecraft/client/renderer/block/model/multipart/MultiPart;)V
+		ARG 1 variants
+		ARG 2 multiPart
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 obj
```
Comment (Harleyoc1) [inline data/net/minecraft/client/renderer/block/model/BlockModelDefinition.mapping]: This isn't required but generally recommend: `other` over `object` for `equals` methods. 
```suggestion
		ARG 1 other
```

File: data/net/minecraft/client/renderer/block/model/ItemOverrides.mapping [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		ARG 2 model
+		ARG 3 modelGetter
+		ARG 4 override
+	METHOD lambda$new$1 (Lit/unimi/dsi/fastutil/objects/Object2IntMap;Lnet/minecraft/client/renderer/block/model/ItemOverride$Predicate;)Lnet/minecraft/client/renderer/block/model/ItemOverrides$PropertyMatcher;
+		ARG 0 predicate
+	METHOD resolve (Lnet/minecraft/client/resources/model/BakedModel;Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/client/multiplayer/ClientLevel;Lnet/minecraft/world/entity/LivingEntity;I)Lnet/minecraft/client/resources/model/BakedModel;
+		ARG 1 model
+		ARG 2 stack
+		ARG 3 level
+		ARG 4 living
```
Comment (Harleyoc1) [inline data/net/minecraft/client/renderer/block/model/ItemOverrides.mapping]: `entity` or `livingEntity`

File: data/net/minecraft/client/renderer/block/model/ItemOverrides.mapping [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+		ARG 5 seed
+	CLASS BakedOverride
+		METHOD <init> ([Lnet/minecraft/client/renderer/block/model/ItemOverrides$PropertyMatcher;Lnet/minecraft/client/resources/model/BakedModel;)V
+			ARG 1 matches
```
Comment (Harleyoc1) [inline data/net/minecraft/client/renderer/block/model/ItemOverrides.mapping]: The field name.
```suggestion
			ARG 1 matchers
```

File: data/net/minecraft/client/renderer/block/model/ItemTransform.mapping [hunk 4]
```diff
@@ -1,7 +1,13 @@
 CLASS net/minecraft/client/renderer/block/model/ItemTransform
+	METHOD <init> (Lcom/mojang/math/Vector3f;Lcom/mojang/math/Vector3f;Lcom/mojang/math/Vector3f;)V
+		ARG 1 rotation
+		ARG 2 translation
+		ARG 3 scale
 	METHOD apply (ZLcom/mojang/blaze3d/vertex/PoseStack;)V
 		ARG 1 leftHand
 		ARG 2 matrixStack
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 obj
```
Comment (Harleyoc1) [inline data/net/minecraft/client/renderer/block/model/ItemTransform.mapping]: See comment about `other` in `equals` above. Again I don't think it's required but generally recommended.

File: data/net/minecraft/client/renderer/block/model/Variant.mapping [hunk 5]
```diff
@@ -1,4 +1,11 @@
 CLASS net/minecraft/client/renderer/block/model/Variant
+	METHOD <init> (Lnet/minecraft/resources/ResourceLocation;Lcom/mojang/math/Transformation;ZI)V
+		ARG 1 modelLocation
+		ARG 2 rotation
+		ARG 3 uvLock
+		ARG 4 weight
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 obj
```
Comment (Harleyoc1) [inline data/net/minecraft/client/renderer/block/model/Variant.mapping]: See previous comments about `obj` in `equals`.

File: data/net/minecraft/client/renderer/block/model/multipart/MultiPart.mapping [hunk 6]
```diff
@@ -1,9 +1,14 @@
...
+	METHOD <init> (Lnet/minecraft/world/level/block/state/StateDefinition;Ljava/util/List;)V
+		ARG 1 definition
+		ARG 2 selectors
 	METHOD bake (Lnet/minecraft/client/resources/model/ModelBakery;Ljava/util/function/Function;Lnet/minecraft/client/resources/model/ModelState;Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/client/resources/model/BakedModel;
 		ARG 1 modelBakery
 		ARG 2 spriteGetter
 		ARG 3 transform
 		ARG 4 location
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 obj
```
Comment (Harleyoc1) [inline data/net/minecraft/client/renderer/block/model/multipart/MultiPart.mapping]: See previous comments about `obj` in `equals`.

File: data/net/minecraft/client/renderer/block/model/multipart/Selector.mapping [hunk 7]
```diff
@@ -1,4 +1,9 @@
 CLASS net/minecraft/client/renderer/block/model/multipart/Selector
+	METHOD <init> (Lnet/minecraft/client/renderer/block/model/multipart/Condition;Lnet/minecraft/client/renderer/block/model/MultiVariant;)V
+		ARG 1 condition
+		ARG 2 variant
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 obj
```
Comment (Harleyoc1) [inline data/net/minecraft/client/renderer/block/model/multipart/Selector.mapping]: See previous comments about `obj` in `equals`.

**Review by Harleyoc1 — CHANGES_REQUESTED** [review]

File: data/net/minecraft/client/renderer/block/model/BlockModelDefinition.mapping (hunk 1 above)
Comment (alcatrazEscapee) [reply data/net/minecraft/client/renderer/block/model/BlockModelDefinition.mapping]: `other` and `object` are fine. `obj` should be changed per [Standards 4.](https://parchmentmc.org/docs/mappings.html)

**Review by Harleyoc1 — APPROVED** [review]

**Review by alcatrazEscapee — APPROVED** [review]

**Pull request merged**